### ISSUE TITLE
Make the Social Ping Notifications Off by Default

### ISSUE THREAD

**Issue by GearBell** [Yellow-Dog-Man/Resonite-Issues#2179]
### Is your feature request related to a problem? Please describe.

The new social status is fine but the ping from them will become problematic for new users and visiting streamers who are unaware of what social statuses are nor that it a setting you can turn off. Those of us with a lot of friends in our lists find its going off a lot too due to the sheer number of contacts.

### Describe the solution you'd like

It might be wise to have the pings off by default, and you can turn them on if you wish. The number of users who want it off might be a bit larger than those who want it on, so it might be best to have it off until the select users choose to turn it on, rather than have everyone immediately go to their settings first thing and turn it off.

### Describe alternatives you've considered

Welcoming new users and streamers then warning them about turning a feature off until they want it, or having to explain to new users the wild pinging they are getting - is at the moment the only alternative, and it is quite awkward.

**Comment by LeCloutPanda** [comment]
Understandably it is a nice feature but being bombarded by notifications is overwhelming and I have been playing this game since launch so I am used to things but as GearBell mentioned if a new user joins and they get bombarded with notifications they make become overwhelmed and become lost as to what to do and where to go and would just end up leaving.

**Comment by Zyzyl** [comment]
Bear in mind that new users don't start with any contacts. Until they start adding people they won't get any notifications at all. Hopefully, by the time they do, one of those contacts could help turn them disabled the notifications if desired.

**Comment by XDelta** [comment]
A brand new user wouldn't be 'bombarded' by these notifications and wouldn't see any of them until adding some contacts in the first place. Existing users who log in may get a decent few extra as they'll already have a number of contacts.

**Comment by LeCloutPanda** [comment]
> Bear in mind that new users don't start with any contacts. Until they start adding people they won't get any notifications at all. Hopefully, by the time they do, one of those contacts could help them disable the notifications if desired.

> A brand new user wouldn't be 'bombarded' by these notifications and wouldn't see any of them until adding some contacts in the first place. Existing users who log in may get a decent few extra as they'll already have a number of contacts.

I understand both of these points but new users will go out and adding everyone then leading to mass notifications and for returning players getting spammed. 

A solution could just be a thing that pops up with the change log that outlines some basics that appears on the home tab or something to mention to the user that new stuff was added and where it was added so people can look for it/disable if need to.

**Comment by Redcyberhawk** [comment]
While first-time users may not be immediately bombarded by the notifications from the newly added Sociable status, I believe that having the option on by default will discourage active contact linking once people start to add more users, and respectively detract from the overall goal of having such a status. While turning off the notification by default would be a recommended solution, I also would like to propose some alternative additions to fix some of the current annoyances with the system:

- Add a configurable setting for Sociable notifications to turn on/off the unique sound, while preserving the notification itself. A lot of the annoyance coming from me personally is the sound playing every so often, which as soft and soothing as it is, can get very annoying to some, even when someone may want to see the notification, but not the sound. We already have controls for other message types with specific types of toasts/audio, so having one for sociable with it's unique sound could be handy.

- Add initial setup/tutorial phrasing for statuses and notification control. This is more an overarching topic that fits the need of a more detailed tutorial world perhaps, but having the initial details of each status, and configuring their relative settings would be a massive boon for new users, and would help to prevent confusion on changing such a feature without having to rely on the community for a fix/guidance.

**Comment by Frooxius** [comment]
We're not going to make it off by default, because that would effectively defeat its purpose and result in a poor discoverability of this feature.

Having people go and opt in into this would mean that most people might not and as a result won't invited into sessions.

I think we're widely overestimating how much would new users get spammed - they don't have nearly as many contacts as established users - meaning fewer users to get this notification from.

It's not really that different from getting other notifications, like messages or invites - they already get those and I don't see this being more confusing.

But even if people feel they get those too much - it's literally just a toggle, it shouldn't be that difficult to keep it on.

**Comment by ShrikeAlvaron** [comment]
Copying a comment I made here as I do think this is worth more consideration should Sociable be kept:

I think closing that issue was perhaps a bit hasty and it's worth considering the concerns about the notification some more. It's pretty evident these days that many people HEAVILY dislike being audibly pinged with notifications they don't care for, you can see the drama that occurs whenever people ping everyone in Discord example (this was especially bad during last MMC if you want to check the channel archive for it...) As such I think the notification would go over much better with people if it was visual-only by default, with the sound disabled but an option provided to enable it for those who wish, as I believe the sound aspect is the side that most bothers people who dislike the notifications being defaulted to on.

Also adding on to that comment, I disagree that having the audible ping disabled by default would defeat the point of the feature.  The visual notification could still be kept on by default, and that itself along with people just manually checking their contacts list would provide ample purpose to the status.  It's simply that when it comes to audio, there's a lot more consideration that needs to be made as some people are very sensitive to such pings for one reason or another.

**Comment by shiftyscales** [comment]
The underlying request in this issue is to disable it outright by default, which is not a suitable solution, which is why this issue was closed, @ShrikeAlvaron.

If you or other users have constructive recommendations/tweaks for how to improve the functionality of this feature- please feel free to open a separate issue, keeping in mine our [reporting requirements](https://github.com/Yellow-Dog-Man/Resonite-Issues#reporting-requirements) to keep things objective, and use neutral language.

E.g. if the notification setting were tweaked to be in line with other notification types that allow the user to choose whether they want the toast, sound, both, or neither.
